docker-app/qfieldcloud/core/views/status_views.py

```diff
@@ -16,17 +18,25 @@ def get(self, request):
         # Try to get the status from the cache
         results = cache.get("status_results", {})
         if not results:
-            results["geodb"] = "ok"
-            # Check geodb
-            if not geodb_utils.geodb_is_running():
-                results["geodb"] = "error"
-
-            results["storage"] = "ok"
-            # Check if bucket exists (i.e. the connection works)
+            # check database
+            results["database"] = "ok"
+            db_conn = connections["default"]
             try:
-                utils.get_s3_bucket()
-            except Exception:
-                results["storage"] = "error"
+                with db_conn.cursor() as cursor:
+                    cursor.execute("SELECT 1")
+            except OperationalError:
+                results["database"] = "error"
+
+            # check storages
+            results["storages"] = {}
+            for storage_key in storages.backends.keys():
+                storage = storages[storage_key]
+
+                try:
+                    results["storages"][storage_key] = storage.check_status()
+                except Exception:
+                    # ManifestStaticFilesStorage may not have check_status method.
+                    continue
```

Comment:
Instead of skipping silently, you can have an entry here so to wonder about a missing key. 

Something like that:

```suggestion
                    results["storages"][storage_key] = "error" 
```